src/api/story.py

```diff
@@ -15,13 +16,21 @@ def get_story(story_id):
         abort(404)
 
 
-@story_api.route("/story")
-def list_story():
-    stories = data_source.get_story_list()
-    return jsonify(stories.dict())
+@story_api.route("/stories", methods=["GET", "POST"])
+def create_or_get_list_story():
+    if request.method == "POST":
+        data = request.get_json()
+        try:
+            saved_data = data_source.create_story(data)
+        except ValidationError as e:
+            return jsonify(e.message), 400
+        return jsonify(saved_data.dict()), 201
+    if request.method == "GET":
+        stories = data_source.get_story_list()
+        return jsonify(stories.dict())
```

Comment:
Can we split this function to `create_story` and `get_list_story` ?

Reply:
Do you mean that need to split the logic for POST and GET requests?

Reply:
Yes